rpm/lzop.spec

```diff
@@ -39,7 +37,7 @@ lzop documentation. Man pages and html docs.
 export CPPFLAGS="-I/usr/include/lzo"
 export LDFLAGS="-L%{_libdir}"
 %configure
-%__make %{?jobs:-j%{jobs}}
+%make_build
 
 %install
 %makeinstall
```

Comment:
Just an observation: so we have both %make_install and %makeinstall? :thinking:

Reply:
Hm, no idea really.

Reply:
%makeinstall is old naming and should not be used in general, it works though.

Reply:
> Just an observation: so we have both %make_install and %makeinstall? thinking

https://github.com/rpm-software-management/rpm/commit/0fddb3cbd80fc763ecacfea5b82631f7693915c2

Reply:
Ok, I get it. We should fix those on sight.